shared/chat/conversation/messages/system/index.js

```diff
@@ -17,7 +18,15 @@ type Props = {
 const SystemNotice = ({channelname, message, onManageChannels, you}: Props) => (
   <UserNotice style={{marginTop: globalMargins.small}} username={message.author} bgColor={globalColors.blue4}>
     <Text type="BodySmallSemibold" backgroundMode="Announcements" style={{color: globalColors.black_40}}>
-      {message.author === you ? 'You' : message.author} {message.message.stringValue()} #{channelname}.
+      {message.author === you ? 'You' : message.author}
+      {' '}
+      {message.message.stringValue()}
+      {' '}
+      #{channelname}
+      {' '}
+      at
+      {' '}
+      {formatTimeForMessages(message.timestamp)}.
```

Comment:
:sheep:  `You joined #channel at Yesterday 4:03 PM.` doesn't sound quite right...think it'd make sense to put it on it's own line like chat timestamps? or maybe a `formatTimeForSystem()`?